docs: DOC-1381: Uris and authentication (#8030)
deephaven/deephaven-core: docs/groovy/how-to-guides/use-uris.md

sidebar_label: URI
---

This guide shows you how to use Deephaven's [URIs](https://deephaven.io/core/javadoc/io/deephaven/uri/package-summary.html) to share tables across server instances and networks.

A URI, short for [Uniform Resource Identifier](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier), is a sequence of characters that identifies a resource on the web. Think of a URI as a generalization of a URL. A Deephaven URI identifies a table on a server instance. By linking to a URI, you can access and work with tables from other Deephaven server instances without needing to replicate the data or queries that created them.

> [!NOTE]

> > URIs can be used to share tables across Groovy and Python instances interchangeably. For how to use URIs in Python, see [the equivalent guide](/core/docs/how-to-guides/use-uris).

Reviewer: ```suggestion
> You can use URIs to share tables across Groovy and Python instances interchangeably. For how to use URIs in Python, see [the equivalent guide](/core/docs/how-to-guides/use-uris).
```

Reviewer: Should apply to python too if accepted.
